### ISSUE TITLE
Google stadia

### ISSUE THREAD

**Issue by Xellerate005** [Genymobile/gnirehtet#419]
Google stadia requires a decent internet connection, wondering if this would help reach those speeds

**Comment by Guilherme7N** [comment]
I use this program on Geforce Now on my phone, it is not Google Stadia but it works very well for me, especially when your pc is connected through an ethernet cable, definitely better than use wi-fi.

**Comment by Xellerate005** [comment]
Yeah I'm wondering if I connect a raspberry pi to ethernet and then use gnirehtet would it make stadia faster, it would be better than buying a long ethernet cable since my router isn't in the best of places

**Comment by Guilherme7N** [comment]
Unfortunately I don't know nothing about raspberry pi, but I wish you luck in your search.
